code/game/objects/structures/mortar.dm

```diff
@@ -198,6 +198,21 @@
 		addtimer(CALLBACK(GLOBAL_PROC, .proc/playsound, T, 'sound/weapons/guns/misc/mortar_travel.ogg', 50, 1), travel_time)
 		addtimer(CALLBACK(src, .proc/detonate_shell, T, mortar_shell), travel_time + 45)//This should always be 45 ticks!
 
+	if(istype(I, /obj/item/binoculars/tactical))
+		var/obj/item/binoculars/tactical/binocs = I
+		playsound(src, 'sound/effects/binoctarget.ogg', 35)
+		if(binocs.set_mortar(src))
+			to_chat(user, "<span class='notice'>You link the mortar to the [I] allowing for remote targeting</span>")
+			return
+		to_chat(user, "<span class='notice'>You unlink the mortar from the [I].")
```

Comment:
no, this is not what is happening.
If binoc.set mortar is false, it seems one mortar was unliked, you never check if it's this one